src/dotnet/APIView/APIViewWeb/MiddleWare/UITestsMiddleWare.cs

```diff
@@ -18,19 +20,20 @@ public UITestsMiddleWare(RequestDelegate next)
 
         public async Task InvokeAsync(HttpContext httpContext, IConfiguration config)
         {
-            var cosmosClient = new CosmosClient(config["Cosmos:ConnectionString"]);
-            var dataBaseResponse = await cosmosClient.CreateDatabaseIfNotExistsAsync("APIView");
+            var cosmosClient = new CosmosClient(config["CosmosEndpoint"], new DefaultAzureCredential());
```

Comment:
Do you really want to use DefaultAzureCredential here? Do we expect to ever use anything other than MI? What is the auth story for local testing?

Reply:
Let me check. I think this is not required to use dummy test instance.

Reply:
I will submit a separate PR to remove this Will file an issue to track it.